docs/pywbemclicmdshelp.rst

```diff
@@ -882,27 +884,23 @@ The following defines the help output for the `pywbemcli connection save --help`
 
 ::
 
-    Usage: pywbemcli connection save [COMMAND-OPTIONS]
+    Usage: pywbemcli connection save [COMMAND-OPTIONS] NAME
 
       Save the current connection as a new WBEM connection definition.
 
       Create a new WBEM connection definition in the connections file from the
-      current connection. A connection definition with that name must not yet
-      exist.
-
-      This command is useful when you have defined a connection on the command
-      line and want to save it in the connections file for later use.
+      current connection with the NAME argument as connection name. A connection
```

Comment:
On " with the NAME argument as connection name":
1. To be precise, it is the connection definition name.
2. Along with my suggestion on "connection add", I suggest to shorten the first sentence to: "Create a new WBEM connection definition named NAME in the connections file from the current connection."